src/test/java/com/lambdatest/cucumber/ParallelAndroidTest1.java

```diff
@@ -1,11 +1,10 @@
 package com.lambdatest.cucumber;
 
 import com.lambdatest.LambdaTestSerenityTest;
-import cucumber.api.CucumberOptions;
+import io.cucumber.junit.CucumberOptions;
 import net.serenitybdd.cucumber.CucumberWithSerenity;
 import org.junit.runner.RunWith;
 
-@RunWith(CucumberWithSerenity.class)
-@CucumberOptions(features = "src/test/resources/features/single.feature")
-public class ParallelAndroidTest1 extends LambdaTestSerenityTest {
+@RunWith(CucumberWithSerenity.class) @CucumberOptions(features = "src/test/resources/features/single.feature") public class ParallelAndroidTest1
```

Comment:
why you want to run just specific feature file.

Reply:
updated